viewer-prototype/src/browser/trace-viewer/components/table-output-component.tsx

```diff
@@ -140,16 +139,12 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
             colIds.push(0);
         }
 
-        columnEntries.forEach(entry => {
-            const id = this.showIndexColumn ? ++entry.id : entry.id;
+        columnEntries.forEach(columnHeader => {
+            const id = this.showIndexColumn ? ++columnHeader.id : columnHeader.id;
```

Comment:
later patch... can we split this... I really hate counting on pre/post increments.